lib/application.js

```diff
@@ -46,6 +46,18 @@ module.exports = class Application extends Emitter {
     this.context = Object.create(context);
     this.request = Object.create(request);
     this.response = Object.create(response);
+    this.customStatuses = {};
+  }
+
+  /**
+   * Add custom status code
+   * @param {Number} code
+   * @param {String} description
+   * @api public
+   */
+
+  addStatus(code, description) {
+    this.customStatuses[code] = description;
```

Comment:
only issue is naming. i'd prefer this to be called `customStatusCodes`. what does everyone else think?

Reply:
other than that, i'm good with this PR

Reply:
Not that my opinion matters, but personally I'd prefer `addCustomStatusCode`, erring on the side of a more descriptive name